core/services/ocr2/plugins/functions/config/config.go

```diff
@@ -31,6 +36,8 @@ type ReportingPluginConfigWrapper struct {
 }
 
 func DecodeReportingPluginConfig(raw []byte) (*ReportingPluginConfigWrapper, error) {
+	fmt.Printf("DecodeReportingPluginConfig Bytes: %x\n", raw)
```

Comment:
Remove here and multiple ones below. I'm assuming those are leftovers from local tests.